privacyidea/static/components/token/views/token.enroll.daypassword.html

```diff
@@ -0,0 +1,76 @@
+<p class="help-block"
+   ng-hide="$state.includes('token.wizard')"
+   translate>The DayPassword token is a time based password token. You can paste a secret key or
+    have the server generate the secret and scan the QR code with an authenticator app that supports the token.
+</p>
+
+<ng-include src="instanceUrl + '/static/components/token/views/token.display.apps.html'">
+</ng-include>
+
+<h4 ng-hide="$state.includes('token.wizard')" translate>Token data</h4>
+
+<div class="form-group">
+    <div ng-hide="checkRight('daypassword_force_server_generate')">
+        <input type="checkbox" ng-model="form.genkey"
+               name="generate" id="generate">
+        <label for="generate" translate>Generate password on the Server</label>
+    </div>
+
+    <div ng-show="form.genkey">
+        <p class="help-block" translate>
+            The server will create the password
+            and a QR Code
+            will be
+            displayed to you to be scanned.
+        </p>
+    </div>
+</div>
+<div class="form-group"
+     ng-show="checkRight('totp_2step=allow') || (!$state.includes('token.wizard') && checkRight('daypassword_2step=force'))">
+    <input type="checkbox" ng-model="form['2stepinit']"
+           ng-disabled="checkRight('daypassword_2step=force')"
+           name="twostep_enrollment" id="twostep_enrollment">
+    <label for="twostep_enrollment" translate>Use two-step enrollment with the privacyIDEA Authenticator App</label>
+</div>
+<div class="form-group" ng-hide="form.genkey">
+    <label for="otpkey" translate>OTP Key</label>
+    <input type="text" ng-pattern="/^[0-9a-fA-F]*$/" id="otpkey"
+           autofocus
+           class="form-control"
+           placeholder="{{ 'Enter the password...'|translate }}"
+           ng-model="form.otpkey" name="otpkey">
+</div>
+<div class="form-group"
+    ng-hide="checkRight('totp_otplen')">
+    <label for="otplen" translate>password length</label>
+    <select class="form-control"
+            id="otplen"
+            ng-model="form.otplen"
+            ng-options="otplen for otplen in formInit.otplens">
+    </select>
+    <p class="help-block" ng-show="form.genkey" translate>
+        The most Authenticator only supports a password length of 6.
+    </p>
+</div>
+<div class="form-group"
+    <label for="timestep" translate>Timestep</label>
+    <input type="text"
+           required
+           class="form-control"
+           placeholder="60"
+           ng-model="form.timeStep">
+    The time that the password can be used in seconds.
+</div>
+
+<div class="form-group"
+    ng-hide="checkRight('daypassword_hashlib')">
+    <label for="hashlib" translate>Hash algorithm</label>
+    <select class="form-control"
+            id="hashlib"
+            ng-model="form.hashlib"
+            ng-options="hashlib for hashlib in formInit.hashlibs">
+    </select>
+    <p class="help-block" ng-show="form.genkey" translate>
+        The Google Authenticator only supports the SHA1 algorithm.
```

Comment:
better be more general:
"Some Authenticator Apps only support specific algorithms"

Reply:
https://github.com/privacyidea/privacyidea/pull/3602/commits/1d33e4a5414b7c3962159450ba40a6e9f9e29488